ansible_collections/arista/avd/molecule/eos_cli_config_gen/documentation/devices/mac-security-eth-po-entropy.md

```diff
@@ -301,7 +301,6 @@ interface Ethernet3
 !
 interface Port-Channel3
    description L2-PORT
-   switchport
    switchport trunk allowed vlan 1-5
```

Comment:
switchport is still removed from port-channel. Please implement same fix as you did for ethernet interfaces.

Reply:
Fixed in commit f0f6597